### ISSUE TITLE
Archive the repo

### ISSUE THREAD

**Issue by UltiRequiem** [pietvanzoen/deno-dotenv#65]
Since this is now part of the std,
https://github.com/pietvanzoen/deno-dotenv#-dotenv-is-now-a-part-of-deno_std-

It would help if this repo is archived, so is more clear that we should prefer using std/dotenv than this.

**Comment by pietvanzoen** [comment]
Makes sense. I've been meaning to drop into the existing issues here and direct people to the std lib. Archiving prevents comments on issues so I'd like to take care of that first.

**Comment by nachoguru** [comment]
Hi @pietvanzoen hope you're well and haven't faded from Open Source. Hopefully you're just on a brief holiday from dealing with the constant demands of random strangers... like me. :)

Rather than open a new issue I thought I would just add to this one. I am new-ish to Github and also to Deno in that I only get to poke at this stuff every second weekend or so.  But my  request is probably something I could do, if you like, as it's just text editing. Let me know if I should give it a shot as my first PR on Github: 

As a noob to Deno and Deno Deploy, and as someone who'd never used Node either, navigating the mazes of old docs around environment variables in multiple ecosystems cost me a half day last month, and I was pretty embarrassed when I finally clued I to how easy it really was... if only I had found the right docs/links first.

It's great that this repo is now `std` but and yes that is in the README but it is not apparent on the leader board here:

![Screenshot_20230113_175505_Firefox](https://user-images.githubusercontent.com/62521616/212435140-c4b5637c-abcc-448c-ac50-37ca719a6e7b.png)

...and I think it should be. Check out media_types just 2 items above it... I strongly feel this edit would help many more new adopters like me.

Never underestimate my -- er, **our** -- ability to get lost in the weeds. :)

Is it time, and is it possible, for this report to look  / act more like media_types ? If so I could give it a shot... see the general differences below.

![Screenshot_20230113_180247_Firefox](https://user-images.githubusercontent.com/62521616/212436157-c094fc17-e533-43d3-b95a-756c08b326fd.png) 

![Screenshot_20230113_180257_Firefox](https://user-images.githubusercontent.com/62521616/212436186-f5bc61ff-d693-4871-8cb5-f2ce82428854.png)

**Comment by pietvanzoen** [comment]
Hey folks. Apologies for going incognito for a while there. My wife and I just had our 2nd child so it's been difficult to prioritise this stuff.

Will archive the project now.

So long and thanks for all the fish.
